sed/loader/flash/dataframe.py

```diff
@@ -257,10 +259,18 @@ def df_train(self) -> pd.DataFrame:
             # contains multiple channels inside. Even though they are resolved per train,
             # they come in pulse format, so the extra values are sliced and individual channels are
             # created and appended to the list
+            aux_alias = self._config.get("aux_alias", "dldAux")
+            sub_channels = self._config.get("aux_subchannels_alias", "dldAuxChannels")
             if channel == "dldAux":
-                aux_channels = self._config["channels"]["dldAux"]["dldAuxChannels"].items()
-                for name, slice_aux in aux_channels:
-                    series.append(pd.Series(dataset[: key.size, slice_aux], index, name=name))
+                aux_channels = self._config["channels"][aux_alias][sub_channels].items()
+                for name, values in aux_channels:
```

Comment:
Why define aux_channels here? Can't you just directly iterate the dict?

Reply:
Yes surely. Just makes the argument really long.